folly/experimental/symbolizer/Symbolizer.cpp

```diff
@@ -189,7 +189,7 @@ void Symbolizer::symbolize(
 
       // Get the unrelocated, ELF-relative address by normalizing via the
       // address at which the object is loaded.
-      auto const adjusted = addr - lmap->l_addr;
+      auto const adjusted = addr - (uintptr_t)lmap->l_addr;
```

Comment:
Let's use C++ casts in C++.
```suggestion
      auto const adjusted = addr - static_cast<uintptr_t>(lmap->l_addr);
```

Reply:
ah nope this is the old fashion caddr_t type we re dealing with here :-) reinterpret_cast works in this case.